src/electron/platform/android/rule-information-provider.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-import { UnifiedResolution } from 'common/types/store-data/unified-data-interface';
+import { UnifiedFormattableResolution, UnifiedResolution } from 'common/types/store-data/unified-data-interface';
```

Comment:
(Can't add a note on line 59, so adding it here)
All references to `UnifiedResolution` should be replaced with `UnifiedFormattedResolution`. Line 59 is still returning a `UnifiedResolution`. In addition, `getColorContrastUnifiedResolution` might be more appropriate as `getColorContrastUnifiedFormattedResolution`